docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,116 @@
+# Release 393 (10 August 2022)
+
+## General
+
+* Add support for the SQL [Merge](/sql/merge) statement. ({issue}`7933`)
+
+## Security
+
+* Fix potential incorrect rejection when using OAuth 2.0 refresh tokens. ({issue}`13168`)
+* Fix showing outdated OAuth 2.0 refresh tokens in the web UI. ({issue}`13168`)
+
+## Docker image
+
+* Add image for `ppc64le`. ({issue}`13522`)
+
+## ClickHouse connector
+
+* Add support for the `unsupported-type-handling` catalog configuration
+  property. ({issue}`13542`)
+* Improve performance for queries with selective joins. ({issue}`13334`)
+
+## Delta Lake connector
+
+* Add support for the SQL [Merge](/sql/merge) statement. ({issue}`7933`)
+* Add support for the `NOT NULL` column constraint. ({issue}`13436`)
+* Fix writing incorrect results when the order of partition columns is different
+  from the order in the table definition. ({issue}`13505`)
+* Fix failure when reading a table which has partition columns renamed by the
+  remote data source. ({issue}`13521`)
+
+## Druid connector
+
+* Improve performance for queries with selective joins. ({issue}`13334`)
+
+## Hive connector
+
+* Add support for the SQL [Merge](/sql/merge) statement. ({issue}`7933`)
+* Improve performance when using query constraints while fetching table
+  statistics. ({issue}`13488`)
+* Fix error when writing to a table with only a `date` data type while using the
+  Hive metastore. ({issue}`13502`)
+
+## Iceberg connector
+
+* Add support for the SQL [Merge](/sql/merge) statement. ({issue}`7933`)
+* Improve performance of read queries on Iceberg v2 tables with
+  deletion-tracking files. ({issue}`13395`)
+* Allow partitioning over columns which include non-alphanumeric characters in
+  their names. ({issue}`12226`)
+* Fix error when querying tables which are empty and contain no table history. ({issue}`13576`)
+
+## Kafka connector
+
+* Fix predicate failure when there is a negative offset value on the
+  `_timestamp` column. ({issue}`13167`)
+
+## Kudu connector
+
+* Add support for the SQL [Merge](/sql/merge) statement. ({issue}`7933`)
+
+## MariaDB connector
+
+* Improve performance for queries with selective joins. ({issue}`13334`)
+
+## MongoDB connector
+
+* Disallow using names past max length when renaming a table. Previously, long
+  names were silently truncated. ({issue}`13073`)
```

Comment:
We only disallow `RENAME` on such names. So:
```suggestion
* Prevent renaming to a table with a name longer than the max length supported by MongoDB.
  Previously, the name was truncated to the max length. ({issue}`13073`)
```

Reply:
It _does_ say "when renaming a table," though I think the wording you're proposing is still clearer. Will update.